src/Observable/disk.jl

```diff
@@ -252,22 +276,34 @@ function compute_structure_factor_flow_all!(
             # read correlations
             m, χ = read_χ(file_in, Λ, label, verbose = false)
 
-            # save frequency mesh
-            if haskey(file_out, "s/$(Λ)/mesh") == false 
+            if static == false 
+                # save frequency mesh 
                 file_out["s/$(Λ)/mesh"] = m
```

Comment:
We should keep checking if the mesh was already saved, otherwise this gives errors for models with multiple non-vanishing structure factor components.

```suggestion
                if haskey(file_out, "s/$(Λ)/mesh") == false
                    file_out["s/$(Λ)/mesh"] = m
                end
```